src/contracts/payloads/PermissionedPayloadsController.sol

```diff
@@ -83,15 +83,9 @@ contract PermissionedPayloadsController is
     public
     override(PayloadsControllerCore, IPayloadsControllerCore)
     onlyPayloadsManager
-    returns (uint40)
+    returns (uint40 payloadId)
```

Comment:
I think better to return payloadId instead of named return (as its how we do it in the rest of the code)